uwblueprint/community-fridge-kw
# style: desktop styling for upcoming dropoffs and dropoff details

## Notion ticket link

[Response Dropoff Dashboard Styling](https://www.notion.so/uwblueprintexecs/Donor-Dashboard-Flow-Desktop-Friendly-5a0dd880f12d469aa2c85155e82de286)



## Implementation description
* Lots of flexbox 
* Fixed some of the padding + colours for badges
* the mobile looks a bit weird on video because I was resizing my screen but you can go into inspect element (mobile) and it should match designs exactly



## Steps to test
1. Check desktop and mobile styling look close to Figma

https://user-images.githubusercontent.com/19617248/144354903-9c54227d-ef0e-4503-9c24-28d756d2dd7d.mp4


https://user-images.githubusercontent.com/19617248/144355006-99f1ca65-7d25-485c-8461-4e4773be0cee.mp4
